pkg/sql/logictest/testdata/logic_test/alter_table

```diff
@@ -2676,7 +2676,7 @@ INSERT INTO t_add_column_not_null VALUES (1), (2), (3)
 statement error pgcode 23502 null value in column "j" violates not-null constraint
 ALTER TABLE t_add_column_not_null ADD COLUMN j INT NOT NULL DEFAULT (NULL:::INT)
 
-statement error pgcode 23502 null value in column "j" violates not-null constraint
+statement error pgcode 23502 .*
```

Comment:
I think you can do better than .*. Even if there are two different possibilities, it's a regexp, you can use `|`